.prettierignore

```diff
@@ -0,0 +1,2 @@
+*.wasm
+*.wat
```

Comment:
Seems like we will always want to ignore `test/runtime` unless we want to run prettier against our tests and have them pass code style by updating the bindgen...!?